micro-ui/web/micro-ui-internals/packages/modules/hrms/CHANGELOG.md

```
@@ -1,3 +1,9 @@
+## [1.9.3] [24-December-2025]
+### 🔄 Configuration Standardization:
+- **Unified MDMS Config**: Updated `createEmployee` and `EditForm` to use the standardized `UserValidation` MDMS config.
+- **Configurability**: Implemented support for `UICONFIG_MODULENAME` global config to dynamically set the MDMS module name (defaults to `commonUiConfig`).
+- **Consistency**: Aligned validation logic across Login, Profile, and HRMS forms.
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)